src/searching.md

```diff
@@ -3,17 +3,16 @@
 <!-- toc -->
 
 Anki's Browse screen and the Filtered Deck feature use a common method
-of searching for specific cards/notes.
+of searching for specific cards/notes. This method is also used in the search bar below "FSRS parameters" field.
 
 ## Simple searches
 
 When you type some text into the search box, Anki finds matching notes
 and displays their cards. Anki searches in all fields of the notes, but
-does not search for tags (see later in this section for how to search
-for tags). Some examples:
+does not search for tags (For information on how to search for tags, see [this section](##tags-decks-cards-and-notes) of the page). Some examples:
```

Comment:
Needs corrections to capitalization and/or punctuation (and the syntax of the link, I think). 

If the text is to be updated, in the manual, "section" is mostly used to refer to Summary/TOC items, and "page" usually refers to another site (like the FAQ). It's defnitely not 100%, but it would be better to stick with it and move toward consistency and coherency. 

But I don't think it is necessary to change the line at all to add this link -- just use the current text: 
`does not search for tags (see [later in this section](#tags-decks-cards-and-notes) for how to search for tags).`

Reply:
```suggestion
does not search for tags (see [later in this section](#tags-decks-cards-and-notes) to search for tags). Some examples:
```